### ISSUE TITLE
How was the neg_sqr_dist formulated? Could you point out the equation in the paper?

### ISSUE THREAD

**Issue by kalyanainala** [WeitaoVan/L-GM-loss#18]
In the tensorflow folder, the python code resnet_model.py has the following lines
" 
     XY = tf.matmul(feat, means, transpose_b=True)
     XX = tf.reduce_sum(tf.square(feat), axis=1, keep_dims=True)
     YY = tf.reduce_sum(tf.square(tf.transpose(means)), axis=0, keep_dims=True)
     neg_sqr_dist = -0.5 * (XX - 2.0 * XY + YY)  
                                                                           "
I have read the paper but couldn't understand how the  neg_sqr_dist = -0.5 * (XX - 2.0 * XY + YY)  equation was formed. Could you point out the equation in the paper which gave -0.5 * (XX - 2.0 * XY + YY)  ?

**Comment by kalyanainala** [comment]
I believe it is the equation 18 that gave the formula for neg_sqr_dist. ((a-b)2 = (a**2 - 2ab + b**2)) where a, b are the xi and mean respectively.
![equation_18](https://user-images.githubusercontent.com/19593887/88145615-8dfb6680-cbaf-11ea-81b1-6107c90ab498.JPG)
If I am right, Why was the covariance matrix not considered in the equation?
